Packs/AzureSecurityCenter/Integrations/MicrosoftDefenderForCloudEventCollector/README.md

```diff
@@ -0,0 +1,53 @@
+XSIAM collector for Microsoft Defender for Cloud alerts.
+This integration was integrated and tested with version xx of Microsoft Defender for Cloud Event Collector
+
+## Configure Microsoft Defender for Cloud Event Collector on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Microsoft Defender for Cloud Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Microsoft Azure Management URL |  | False |
+    | ID |  | True |
+    | Token |  | True |
+    | Key |  | True |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates &amp;amp; secrets" page of the app. | False |
```

Comment:
Need to fix &amp;amp;
What is this supposed to be?

Reply:
fixed Thanks.